Index: helper/MysqlDatabase.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?php\r\n\r\nclass MysqlDatabase{\r\n    private $connection;\r\n\r\n    public function __construct($servername, $username, $password, $dbname){\r\n        $conn = new mysqli(\r\n            $servername,\r\n            $username,\r\n            $password,\r\n            $dbname,\r\n            3306\r\n        );\r\n\r\n        if ($conn->connect_errno) {\r\n            die(\"Connection failed: \" . $conn->connect_error);\r\n        }\r\n        $this->connection = $conn;\r\n    }\r\n\r\n    public function query($sql){\r\n        $result = $this->connection->query($sql);\r\n        return $result->fetch_assoc();\r\n    }\r\n\r\n    public function execute($sql){\r\n        $result = $this->connection->query($sql);\r\n        return $result;\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/helper/MysqlDatabase.php b/helper/MysqlDatabase.php
--- a/helper/MysqlDatabase.php	(revision 79d81f2c5432ae2c1d9176b95e022de44b3d7616)
+++ b/helper/MysqlDatabase.php	(date 1625361257608)
@@ -9,7 +9,7 @@
             $username,
             $password,
             $dbname,
-            3306
+            3307
         );
 
         if ($conn->connect_errno) {
